qiita_pet/handlers/api_proxy/studies.py

```diff
@@ -242,8 +246,20 @@ def study_files_get_req(study_id, prep_template_id, artifact_type):
     # because selected is initialized to the empty list
     file_types.insert(0, (first[0], first[1], selected))
 
+    # Create a list of artifacts that the user has access to, in case that
+    # he wants to import the files from another artifact
+    user = User(user_id)
+    artifact_options = []
+    for study, artifacts in viewitems(user.user_artifacts(
```

Comment:
This makes sense for most users but for an admin you will like to see not only your artifacts but also the artifacts that belong to the owner of the study you are checking. For example, let's say that you are helping a user and you added an artifact in prep X, then you want to reuse it for prep Y, with this structure you can't.